From 81d584b9027b2b2ddd3209c1582c9ec73c26cc3e Mon Sep 17 00:00:00 2001
From: Simon Wells <simonrwells@gmail.com>
Date: Sun, 27 May 2018 13:44:41 -0700
Subject: [PATCH] Change TR_CURL_SSL_VERIFY to TR_CURL_SSL_NO_VERIFY

use secure by default and change the env var to match curl -k behaviour

Closes: #179
---
 libtransmission/web.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libtransmission/web.c b/libtransmission/web.c
index 997a151b5..ce41e342a 100644
--- a/libtransmission/web.c
+++ b/libtransmission/web.c
@@ -392,7 +392,7 @@ tr_webThreadFunc (void * vsession)
   web->taskLock = tr_lockNew ();
   web->tasks = NULL;
   web->curl_verbose = tr_env_key_exists ("TR_CURL_VERBOSE");
-  web->curl_ssl_verify = tr_env_key_exists ("TR_CURL_SSL_VERIFY");
+  web->curl_ssl_verify = !tr_env_key_exists ("TR_CURL_SSL_NO_VERIFY");
   web->curl_ca_bundle = tr_env_get_string ("CURL_CA_BUNDLE", NULL);
   if (web->curl_ssl_verify)
     {
-- 
2.17.0

